apps/web/app/scripts/fetch-posts.js

```diff
@@ -0,0 +1,97 @@
+import fs from 'fs';
+import path from 'path';
+import { fileURLToPath } from 'url';
+
+/**
+ * Fetches all posts from the CMS by iterating through all pages
+ */
+async function fetchAllPosts() {
```

Comment:
These scripts should probably live a directory up since they are not part of `app`

Reply:
I'll move in next commit